index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <main>
+
+        <nav id="nav-bar">
+            <div class="left-nav-bar">
+                <img class="menu-logo" src="https://img.icons8.com/material-rounded/24/000000/menu.png" />
+                <img class="YouTube-logo" src="https://img.icons8.com/fluent/48/000000/youtube-play.png" />
+                <h3 class="title">YouTube</h3>
+            </div>
+            <div class="mid-nav-bar">
+                <input type="text">
+                <button>Search</button>
```

Comment:
An awesome job using the button tag for the search button. But just like the original, you should use an icon instead of the word **search**

### YOURS
![search name](https://user-images.githubusercontent.com/32361963/87135055-33eecd00-c2a2-11ea-8a98-3e2a98991511.jpg)

### ORIGINAL
![search button original](https://user-images.githubusercontent.com/32361963/87135146-541e8c00-c2a2-11ea-8f36-316813ae3163.jpg)